dashboard/app/models/script.rb

```diff
@@ -607,12 +609,14 @@ def csf_finish_url
     end
   end
 
-  def summarize
+  def summarize(user = nil)
```

Comment:
I'll move peer reviews performed to summarize_user_progress. I'll keep the peer reviews required here though since its a property of the script